docker-compose.yml

```diff
@@ -1,20 +1,37 @@
-broker:
-  image: rabbitmq:3-management
-  ports:
-    - "15672:15672"
-image:
-  build: .
-web:
-  image: lintreview_image
-  command: bash -c 'gunicorn -c ${LINTREVIEW_SETTINGS} lintreview.web:app'
-  environment:
-    LINTREVIEW_GUNICORN_BIND: '0.0.0.0:5000'
-  ports:
-   - "5000:5000"
-  links:
-   - "broker:broker"
-worker:
-  image: lintreview_image
-  command: celery -A lintreview.tasks worker
-  links:
-   - "broker:broker"
+version: '2'
+services:
+  broker:
+    image: rabbitmq:3-management
+    ports:
+      - "15672:15672"
+    networks:
+      - back-tier
+  image:
+    build: .
+  web:
+    image: lintreview_image
+    command: bash -c 'gunicorn -c /code/settings.py lintreview.web:app'
```

Comment:
Looks like it isn't. I just copied what already existed.
Fixed in https://github.com/markstory/lint-review/pull/90/commits/2097805df941af393c100990c3774c05a0a5a3f8